packages/core/src/indexing-store/sqlite/store.ts

```diff
@@ -763,4 +807,12 @@ export class SqliteIndexingStore implements IndexingStore {
     );
     return result;
   };
+
+  private idColumnComparator = ({
+    tableName,
+    schema,
+  }: {
+    tableName: string;
+    schema: Schema | undefined;
+  }) => (schema?.tables[tableName]?.id.type === "bytes" ? "like" : "=");
```

Comment:
Was curious about this - found that like Postgres has the `ILIKE` comparator which is case-insensitive, but for SQLite `LIKE` is case insensitive, and you can configure it (https://sqlite.org/forum/info/af7978156ce072e7e1060f0741dc10d33dc7ccaf702f6d67c5ab692bffa25174). I'm guess you found the same.

Reply:
Yes, there is basically a mirror of this function in the postgres folder